addons/mrp/tests/test_stock_report.py

```diff
@@ -204,6 +204,57 @@ def test_report_forecast_3_report_line_corresponding_to_mo_highlighted(self):
                 else:
                     self.assertFalse(line['is_matched'], "A line of the forecast report not linked to the MO shoud not be matched.")
 
+    def test_kit_packaging_delivery_slip(self):
+        superkit = self.env['product.product'].create({
+            'name': 'Super Kit',
+            'type': 'consu',
+            'packaging_ids': [(0, 0, {
+                'name': '6-pack',
+                'qty': 6,
+            })],
+        })
+
+        compo01, compo02 = self.env['product.product'].create([{
+            'name': n,
+            'type': 'product',
```

Comment:
this has been replaced with:

`'is_storable': True,`

As of a week or 2 ago. Yay refactorings :partying_face: